pattern-lab/source/_patterns/01-molecules/05-HeroComponent/00-hero-component.scss

```diff
@@ -35,49 +35,52 @@
   }
 }
 
-.hero_banner_wrapper {
-  width: 100%;
-  min-height: auto;
-  background-color: #ffffff;
-  background-position: center;
-  background-repeat: no-repeat;
-  background-size: 100% !important;
-  height: auto !important;
-  position: relative;
-  @include clearfix;
-
-  @include respond(tab) {
-    min-height: 540px;
-    height: 580px !important;
-  }
 
-  img {
+.paragraph--type--card {
+  .hero_banner_wrapper {
     width: 100%;
-    height: 100%;
-    object-fit: cover;
-  }
-  div {
-    color: #ffffff;
-  }
-}
-.hero_banner-body {
-  font-family: $bold-font;
-  cursor: grab;
-  left: 0;
-  bottom: 0;
-  height: auto;
-  padding: 20px 70px 20px 70px !important;
-
-  &.center {
-    text-align: center;
+    min-height: auto;
+    background-color: #ffffff;
```

Comment:
all below lines can be part of **background** property,
background-color: #ffffff;
background-position: center;
background-repeat: no-repeat;
background-size: 100%;